.github/workflows/generate_ipynb_files.yml

```diff
@@ -13,6 +14,9 @@ jobs:
       GH_SERVICE_ACCOUNT_NAME: "tflite-bot"
       GH_SERVICE_ACCOUNT_EMAIL: "tflite-bot@antmicro.com"
       GH_SERVICE_ACCOUNT_TOKEN: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+      PREPARE_BUILD_SH: |
+        echo -e "git clone https://github.com/antmicro/tensorflow-arduino-examples.git ./ \n" > build.sh
```

Comment:
You're actually overwriting build.sh here.

Reply:
`./` -> `.`

But I DON'T think it works. You can't just clone to `.` if it's a non-empty directory.

Reply:
right, changed default value of TENSORFLOW_PATH in build.sh, so we can clone this repo to tensorflow-arduino-examples